modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -344,7 +350,10 @@ def minutes_duration(minutes_duration, type)
         end
 
         def va?(type)
-          type == APPOINTMENT_TYPES[:va]
+          [APPOINTMENT_TYPES[:va],
+           APPOINTMENT_TYPES[:va_video_connect_gfe],
+           APPOINTMENT_TYPES[:va_video_connect_atlas],
+           APPOINTMENT_TYPES[:va_video_connect_home]].include?(type)
```

Comment:
Expanded the definition of what a va appointment is to include video appointments, there seems to be a lot of pre-existing issues cropping up from v0 so just mirrors the v0 logic no longer seems like a safe bet and instead including the data to be use as needed seems more important.

Reply:
IDK if you were inspired to do this by looking at the front end code, but this is consistent with what the web is doing:
```
function getAppointmentType(appt) {
  if (appt.kind === 'cc' && appt.start) {
    return APPOINTMENT_TYPES.ccAppointment;
  }
  if (appt.kind === 'cc' && appt.requestedPeriods?.length) {
    return APPOINTMENT_TYPES.ccRequest;
  }
  if (appt.kind !== 'cc' && appt.requestedPeriods?.length) {
    return APPOINTMENT_TYPES.request;
  }

  return APPOINTMENT_TYPES.vaAppointment;
}
```

However, comparing that to how we set appointment type in this adapter, I think we'd be wise to update getAppointmentType to have an else clause that sets the type to "va". It's probably a non-issue because we've probably accounted for all the types, but it's more consistent with what the front end is doing and it will prevent errors if we ever do encounter other types. As it's written now, any unexpected types would be set to nil.

Reply:
good idea. done.